Pinta.Core/Classes/BaseTool.cs

```diff
@@ -46,7 +46,7 @@ public abstract class BaseTool
 	private string ANTIALIAS_SETTING => $"{GetType ().Name.ToLowerInvariant ()}-antialias";
 	private string ALPHABLEND_SETTING => $"{GetType ().Name.ToLowerInvariant ()}-alpha-blend";
 
-	protected static PointI point_empty = new (-500, -500);
+	protected static readonly PointI point_empty = new (-500, -500);
```

Comment:
Perhaps for a future PR, this `point_empty` could be replaced by nullable types?